VTEX - Antifraud Provider API.json

```diff
@@ -1175,72 +791,43 @@
                         }
                     }
                 },
-                "deprecated": false,
-                "servers": [
-                    {
-                        "url": "http://{{providersApiEndpoint}}.com",
-                        "description": "VTEX server URL"
-                    }
-                ]
+                "deprecated": false
             }
         },
         "/transactions/{transactions.Id}": {
             "delete": {
                 "tags": [
-                    "Antifraud  Flow"
+                    "Anti-fraud Flow"
                 ],
-                "summary": "Stop Antifraud Analysis (optional)",
-                "description": "This endpoint is used to notify whether the order has been canceled by the store or the buyer before completing the antifraud analysis status.",
+                "summary": "Stop Anti-fraud Analysis (optional)",
+                "description": "Notifies whether the order has been canceled by the store or the buyer before completing the anti-fraud analysis status.\r\n\r\n## Permissions\r\n\r\nThis endpoint does not require [authentication](https:\/\/developers.vtex.com\/docs\/guides\/authentication) or [permissions](https:\/\/help.vtex.com\/en\/tutorial\/license-manager-resources--3q6ztrC8YynQf6rdc6euk3).",
```

Comment:
Se esse endpoint não requer authentication, por que tem security nele?

Reply:
Adicionado nova descrição, conforme conversamos: ## Permissions\r\n\r\nThis endpoint does not require [permissions](https:\/\/help.vtex.com\/en\/tutorial\/license-manager-resources--3q6ztrC8YynQf6rdc6euk3).